Document common use cases. (#11)
altipla-consulting/logging: README.md

  field2: 3,
})
```

> ### Logging common fields

Reviewer: ```suggestion
### Log common fields
```

> This allows adding common fields to all log messages. It can be useful, for example, to add the current user to all log messages.

Reviewer: Dices this, pero no hay ningún this de nada al que referirse.

```suggestion
Create a custom logger to group fields common to all log messages. It can be useful for example to add the current user inside all messages.
```

```ts
import { logger } from '@altipla/logging'

let userLogger = logger.child({ user: 'user' })

userLogger.info('my message')
userLogger.error('other message')
```

> ### Changing the log level

Reviewer: ```suggestion
### Change the log level
```

> By default the log level is `debug`. You can change it by passing a `level` field to a new child logger.

Reviewer: You no, no le tienes esa confianza al usuario. Y changed by no es lo que quieres decir.

```suggestion
By default the log level is `debug`. It can be changed creating a new child logger with a `level` field configured:
```
